utils/swift_build_support/swift_build_support/debug.py

```diff
@@ -14,29 +14,31 @@
 #
 # ----------------------------------------------------------------------------
 
+from __future__ import absolute_import
 from __future__ import print_function
 
-import subprocess
 import sys
 
-
-def _output(args):
-    try:
-        out = subprocess.check_output(args, stderr=subprocess.PIPE)
-        return out.rstrip().decode()
-    except subprocess.CalledProcessError:
-        return None
+from . import shell
 
 
 def print_xcodebuild_versions(file=sys.stdout):
     """
     Print the host machine's `xcodebuild` version, as well as version
     information for all available SDKs.
     """
-    print(u'{}\n'.format(_output(['xcodebuild', '-version'])), file=file)
-    print(u'--- SDK versions ---', file=file)
-    print(u'{}\n'.format(_output(['xcodebuild', '-version', '-sdk'])),
-          file=file)
+    version = shell.capture(
+        ['xcodebuild', '-version'],
+        dry_run=False, echo=False, optional=True).rstrip()
```

Comment:
I Agree.
Moreover, we need non-Darwin version of `--show-sdks` implementation.
In this PR, I will just omit `, optional=True`